kafka-rest/src/test/java/io/confluent/kafkarest/integration/AuthorizationErrorTest.java

```diff
@@ -46,15 +50,15 @@
 import org.apache.kafka.common.resource.ResourceType;
 import org.apache.kafka.common.security.auth.SecurityProtocol;
 import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import org.apache.kafka.metadata.authorizer.StandardAuthorizer;
+import org.apache.kafka.server.config.ServerConfigs;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.TestInfo;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 import scala.Option;
 
-@Disabled("Until we fix KNET-16472, this test should be disabled")
```

Comment:
Is this part of zk removal? Or merged from some other commit?

Reply:
These changes were merged in master but not in 8.0.x. These are required for removing all zk references in 8.0.x.
cc: @xuliang2019